uqcs/admin.py

```diff
@@ -151,6 +152,24 @@ def delete(s, admin_user, member_id):
     s.delete(user)
     return redirect("/admin/accept", 303)
 
+@admin.route('/dump_members')
+@needs_db_and_admin
+def dump_members(s, admin_user):
+
+    members = s.query(m.Member).filter(m.Member.paid != None).all()
+    data = io.BytesIO()
+
+    for mem in members:
+        fn = mem.first_name
+        ln = mem.last_name
+        em = mem.email
+        data.write(",".join([fn, ln, em]).encode('utf-8') + b"\r\n")
```

Comment:
Union also ask for student numbers, so we should include that (and also indicate non students)

Reply:
Good point, I'll add that in